### PR TITLE
refactor: fix ripple-renderer in TS3.7

### PR DESCRIPTION
**Pull request by mprobst**
Another DOM style nullable issue.

### CODE REVIEW

**Comment by mprobst** [comment]
Paul: usually we transition to TypeScript version++ within a few weeks, so
that might not be worth it for this instance. It might be useful if you
intend to support the old version for a longer period of time. It might
still pay off if you're looking to build generic infrastructure that could
be useful for each language upgrade, not sure.

**Comment by devversion** [comment]
@mprobst Yeah that's fair. It shouldn't be much effort to have a job for that though. I don't know.. probably not a priority either.

File: src/material/core/ripple/ripple-renderer.ts [hunk 1]
```diff
@@ -147,7 +147,8 @@ export class RippleRenderer {
     ripple.style.width = `${radius * 2}px`;
 
     // If the color is not set, the default CSS color will be used.
-    ripple.style.backgroundColor = config.color || null;
+    // TODO(TS3.7): Type 'string | null' is not assignable to type 'string'.
+    ripple.style.backgroundColor = (config.color || null) as any;
```
Comment (devversion) [inline src/material/core/ripple/ripple-renderer.ts]: After thinking more about it, wouldn't it be better to fix it by doing?

```suggestion
    ripple.style.backgroundColor = config.color || '';
```

**Comment by mprobst** [comment]
We generally only do the most conservative fix when doing language
upgrades, otherwise we end up spending all our time fighting e.g. mocked
tests that check for precise values.

Your fix is appropriate though, I think. Maybe in a follow up PR?

**Comment by devversion** [comment]
Yeah totally reasonable. Sounds good to me. That helped understanding the process.

**Pull request merged**